core/StaticArray.carp

```diff
@@ -1,5 +1,23 @@
 (defmodule StaticArray
 
+  (doc map! "Maps a function over the static array `xs`, mutating it in place. The difference to Array.endo-map (which does the same thing internally) is that this function takes a ref (since you can never have static arrays as values) and that it returns ().")
+  (defn map! [xs f]
```

Comment:
For convenience, could we return `xs` here anyway?

Reply:
I don't think that's very convenient since you'll have to `(ignore ...)` that often, no?

Reply:
Don't believe so as every static array literals are (Ref StaticArray)

Reply:
Hm, okay! I just thought that a `map!` that doesn’t return anything looks a little weird, but I guess that makes sense.

Reply:
Since all other `!` function returns `()` I think this one should too.

Reply:
Sure, this can be marked as resolved from my end :)